scripts/run_app.sh

```diff
@@ -24,7 +24,7 @@ fi
 
 # Output the current git revision
 if [ -z "$EQ_GIT_REF" ]; then
```

Comment:
There was also discussion about it being somewhere in the log output at startup for Splunk, is it logged anywhere?